posthog/models/entity.py

```diff
@@ -48,3 +47,26 @@ def to_dict(self) -> Dict[str, Any]:
             "math_property": self.math_property,
             "properties": [prop.to_dict() for prop in self.properties],
         }
+
+    def equals(self, other) -> bool:
```

Comment:
Would appreciate a look here: unsure if there should be anything else inside here, or if this is too much, for two entities to be considered equal

Reply:
I think this can work. Math checks are unnecessary iirc. Funnels shouldn't need to use any of those math properties they should just be counting persons going through the funnel. Otherwise, adding some tests and this should be good